src/Matrix.jl

```diff
@@ -373,7 +373,7 @@ the submatrix $A$ of $M$ defined by `A[i,j] = M[rows[i], cols[j]]`
 for `i=1,...,length(rows)` and `j=1,...,length(cols)`.
 Instead of a vector, `rows` and `cols` can also be:
 * an integer `i`, which is  interpreted as `i:i`, or
-* `:`, which is interpreted as `1:nrows(M)` or `1:ncols(M)` respectively.
+* `:`, which is interpreted as `1:number_of_rows(M)` or `1:number_of_columns(M)` respectively.
```

Comment:
```suggestion
* `:`, which is interpreted as `1:nrows(M)` or `1:ncols(M)` respectively.
```

Reply:
I would like to change this back

Reply:
ok. I add a ci skip to this to remove the waiting time, and this should not change anything for correctness

Reply:
ok thanks